### DISCUSSION TITLE
Npc transmog : Warpweaver Ta' Oren don't have the option ''Transmog'' HELP!!!

### DISCUSSION THREAD

**Discussion by GalaxyMS** [TrinityCore/TrinityCore#26986] [Q&A]
Seens like Ta'oren don't have the Transmog option in the conversation menu, someone help me...
I love the new Transmog System on 9.1 and i can't get it, thanks.
![WoWScrnShot_100121_110256](https://user-images.githubusercontent.com/91585847/135634345-d18ace60-7ff1-4d16-bc5d-c58fa3f633c7.jpg)

**Comment by Aokromes** [comment]
update your core

**Reply by GalaxyMS** [reply]
thanks, how can i update my core files ?
